tests/suites/test_suite_psa_crypto.function

```diff
@@ -2347,6 +2347,89 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void hash_compute_fail( int alg_arg, data_t *input,
+                        int output_size_arg, int expected_status_arg )
+{
+    psa_algorithm_t alg = alg_arg;
+    uint8_t *output = NULL;
+    size_t output_size = output_size_arg;
+    size_t output_length = INVALID_EXPORT_LENGTH;
+    psa_status_t expected_status = expected_status_arg;
+    psa_status_t status;
+
+    ASSERT_ALLOC( output, output_size );
+
+    PSA_ASSERT( psa_crypto_init( ) );
+
+    status = psa_hash_compute( alg, input->x, input->len,
```

Comment:
ITYM `psa_hash_compare`. Indeed, I added some test cases.

Reply:
I think I meant `psa_hash_verify` because I was mislead by the title of this PR, the first commit message, and the name of the next test function: `hash_compute_verify`.